### ISSUE TITLE
Add support for `end` keyword.

### ISSUE THREAD

**Issue by nsillik** [sjbarag/brs#200]
The `END` keyword generates the error "Found unexpected token 'END'". This is a valid keyword and should be allowed.

**Comment by TwitchBronBron** [comment]
Can you provide a code sample showing how you're using `end`?

**Comment by nsillik** [comment]
I'm currently just using it to kill the app in certain places
```brightscript
if true then
  END ' valid
end if
```

I'm pretty sure its usage is the same as `stop` (#192).

**Comment by sjbarag** [comment]
Fascinating!  I've never seen this.  @nsillik do you happen to have a link to the docs that describes this?  It'd be super handy when validating :smile: 

Thanks for the report!

**Comment by nsillik** [comment]
The docs mention it as a keyword, but it doesn't say how it's to be used, unfortunately. I'm actually considering getting rid of it. I'm pretty sure that as a bare keyword (i.e. not a part of an `end sub` or `end if`, etc.) that it's the same as `stop`

**Comment by vbasky** [comment]
https://sdkdocs.roku.com/display/sdkdoc/Reserved+Words

**Comment by TwitchBronBron** [comment]
Here's the documentation on exactly what "end" does.

> *END*
> Terminates execution normally.

https://sdkdocs.roku.com/display/sdkdoc/Program+Statements#ProgramStatements-END

Just ran this test, and it seems to terminate the program. I don't get the debugger prompt or anything through telnet.

```brightscript
function main()
    for i = 0 to 10 step 1
        print i
        stop
    end for
end function
```

**Comment by TwitchBronBron** [comment]
@sjbarag I'm working on this.

**Comment by TwitchBronBron** [comment]
I forgot, I already fixed this in #211. This is working now in the latest codebase.

**Comment by sjbarag** [comment]
Oh right!  Thanks @TwitchBronBron :smile: 

Closed by #211
